### ISSUE TITLE
Where is the Feature rendering part in uploaded code?

### ISSUE THREAD

**Issue by jhq1234** [pfnet-research/distilled-feature-fields#10]
Hi, Thanks for uploading this great work. I really enjoy it.

I got pre-trained Lseg model successfully and followed your code tutorial. 

Calculating the feature loss part, I can't find the feature rendering part mentioned in the original paper.
![Selection_648](https://user-images.githubusercontent.com/68729992/225841987-71ed977e-822c-4460-af90-a4be0b52b4a7.png)

It seems that in the code, the points are casted on the object surface with 5 points, and then this encoded by neural network. And then, these outputs are directly used by calculating loss. Are they corrects?

**Comment by soskek** [comment]
Sorry for the delay in noticing this issue.

This repository is for simple and efficient training via instant NGP (based on ngp_pl repository). So, some parts are different from the paper. The difference includes feature rendering.
Instead of volume rendering of features (it requires much memory), this repo uses a few points on the surface.

https://github.com/pfnet-research/distilled-feature-fields/issues/5#issuecomment-1332039782

>First, I'm sorry that this part is not fully implemented in this NGP-based codebase to keep the simplicity of the cuda code and make it much more memory efficient in the NGP framework. This repository is a specific variant of the DFF model and has some differences from the paper's one.
Rendering a feature via volume rendering only uses a tiny set of samples (5 points) around the rendered surface. The code is [here](https://github.com/pfnet-research/distilled-feature-fields/blob/master/models/rendering.py#L188-L192). This is simple but may lead to worse results sometimes.
